JetStream documentation (#14099)
kyma-project/kyma: docs/05-technical-reference/00-architecture/evnt-01-architecture.md

## Eventing Controller

Eventing Controller manages the internal infrastructure in order to receive an event. It watches Subscription Custom Resource Definitions. When an event is received in an Application, it lays down the Eventing infrastructure in NATS in order to trigger a Function. Eventing Controller also dispatches messages to subscribers such as a Function or another workload.

## JetStream

> Kyma now supports JetStream, which is a persistence offering from NATS, that guarantees `atleast once` delivery. It is build-in within our default NATS backend and can be enabled as shown [here](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#enable-jet-stream).

Reviewer: ```suggestion
Kyma now supports JetStream, which is a persistence offering from NATS, that guarantees `at least once` delivery. It is built-in within our default NATS backend. Follow the [tutorial](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#enable-jet-stream) to enable Kyma with JetStream.
```

> Kyma now supports JetStream, which is a persistence offering from NATS, that guarantees `at least once` delivery. It is built-in within our default NATS backend. Follow the [tutorial](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#enable-jet-stream) to enable Kyma with JetStream.

Reviewer: ```suggestion
Kyma now supports JetStream, which is a persistence offering from NATS, that guarantees at-least-once delivery. It is built-in within our default NATS backend. Follow the [tutorial](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#enable-jet-stream) to enable Kyma with JetStream.
```

The key advantages of JetStream over Core NATS are:

> - [At least once delivery of JetStream](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#at-least-once-delivery-guarantees-using-jet-stream-backend) vs at most once delivery of NATS.

Reviewer: ```suggestion
- [At least once delivery of JetStream](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#at-least-once-delivery-guarantees-using-jet-stream-backend) compared to at most once delivery of NATS.
```

> - [At least once delivery of JetStream](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#at-least-once-delivery-using-jet-stream-backend) compared to at most once delivery of NATS.

Reviewer: ```suggestion
- [At-least-once delivery of JetStream](../../04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md#at-least-once-delivery-using-jet-stream-backend) compared to at most once delivery of NATS.
```
